### ISSUE TITLE
[Bug] BetterNether Wood Signs do not drop as items when broken after being placed

### ISSUE THREAD

**Issue by genram** [quiqueck/BetterNether#151]
### What happened?

After updating to 1.20 Fabric/ 9.0.7 Better Nether / 3.0.1.0 BCLib  from 1.19.4, all existing Signs made from Better Nether woods (Sakura, Reed, Mushroom ect) became invisible. 

Right Clicking them allowed them to be edited and made them Visible again, but the ones on the wall now look like they are on the ground instead (sticks showing and floating in the air away from the wall). When you break them they do not drop as an item. Newly created signs will place on the wall fine but still drop nothing when broken. 

Vanilla signs (oak, spruce ect) work fine.

I have tried this in a single player world with only Better Nether / Better End/ and BCLib in the mods folder and get the same result.

### BetterNether

9.0.7

### BCLib

3.0.1.0

### Fabric API

0.84.0

### Fabric Loader

0.14.21

### Minecraft

1.20.1

### Other Mods

```shell
Arrow Entity Loot Drop
Axolotl Item Fix
Bedrockify
Better End
Cloth Config
Continents
Crawl
Deeper Oceans
Elytraslot
EMI
Grass Overhaul
Loyal Items
More Axolotl Variants
Mo Glass
Modmenu
Trinkets
Yosbr
Yung's API
Yung's Better - Desert Temples, Mineshafts, Netherfortresses, Ocean Monuments, Strongholds, Witchhuts, Extras
```

**Comment by quiqueck** [comment]
Did the world-patcher run when you first updated your world to 1.20? The way signs are handled was changed for 1.20 and the patcher is supposed to fix that.

**Comment by quiqueck** [comment]
Nevermind, the sign patcher actually never made it to the release version. Will fix that. When you open the world with the next BCLib version, the Patch system should take care of the issue.

**Comment by quiqueck** [comment]
This should be fixed with BetterNether 9.0.8. Please re-open the issue if the problem was not solved for you.
